lms/static/sass/_developer.scss

```diff
@@ -79,4 +79,140 @@
   .global-new, #global-navigation {
     display: none;
   }
+
+  // Copied from _pagination.scss in cms
+  .pagination {
```

Comment:
@frrrances New styles for the paging footer will need to be a part of this PR before it can merge. The <a> tags in for the arrows were changed to <button> tags as part of the accessiblity review, which will break the look of the footer in studio if we merge as is.